km61/Chala_Hanna/laboratory1.sql

```diff
@@ -1,2 +1,120 @@
 -- LABORATORY WORK 1
 -- BY Chala_Hanna
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+змінювати структуру таблиць
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER CHALA IDENTIFIED BY 123
```

Comment:
`CREATE USER CHALA IDENTIFIED BY 123`

Password should contain alphabetical symbols